### PR TITLE
update change log

### PR DESCRIPTION
**Pull request by michmuel**
- add a note related to the approach how packages are imported
- extend the information for version 2.4.7

### CODE REVIEW

**Review by voisardf — APPROVED** [review]
thanks

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+Notes:
+------
+- This python package specifies the version numbers only for directly imported python packages. This approach may result in a build failure of the package if there arise incompatibilities between imported packages over time. The build process of the master branch is regularely tested in an automatic process.
```
Comment (jwkaltz) [inline CHANGES.rst:6]: Can we define "of the package" -> "of older versions of the project"

Comment (jwkaltz) [inline CHANGES.rst:6]: regularly

**Review by jwkaltz — APPROVED** [review]

**Review by jwkaltz — APPROVED** [review]
Since we suggest that build might fail over time, we could also suggest how to work in a productive environment (create docker images for your images which you tag and do not delete as long as they are used in production)

**Comment by michmuel** [comment]
> Since we suggest that build might fail over time, we could also suggest how to work in a productive environment (create docker images for your images which you tag and do not delete as long as they are used in production)

That's a good point. We could address it at the next user meeting.

**Pull request merged**